subprojects/resources-gcs/src/integTest/groovy/org/gradle/integtests/resource/gcs/fixtures/stub/HttpStub.groovy

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.resource.gcs.fixtures.stub
+
+import groovy.transform.ToString
+
+@ToString
+class HttpStub {
```

Comment:
Given that so many test fixtures are duplicated, we should consider hoisting these to the `internal-integ-testing` module.